storybook/stories/Components/Filters/RangeFilter.stories.tsx

```diff
@@ -41,45 +41,98 @@ const RangeFilterStory = ({title, filterGroup, onChange, ...props} : {
   </>
 );
 
-const filterGroupConfig: TFilterGroupConfig = {
-    rangeTypes: [{
-        key: 'this.field',
-        name: 'Years',
-    }],
-    min: 1,
-    max: 2,
-}
-
-const filerGroup: IFilterGroup = {
-    type: VisualType.Range,
-    field: 'this.field',
-    config: filterGroupConfig,
-    title: 'title_filter_group'
-}
-
-const onChangeTypeStory: onChangeType = () => null
-
-
 export const TermFilterStoryMore = RangeFilterStory.bind({});
 TermFilterStoryMore.args = {
     title: 'RangeFilter',
-    filterGroup: filerGroup,
+    filterGroup: {
+      type: VisualType.Range,
+      field: 'this.field',
+      config: {
+        rangeTypes: [{
+            key: 'this.field',
+            name: 'Years',
+        }],
+        min: 1,
+        max: 2,
+    },
+      title: 'title_filter_group'
+  },
     filters: rangeFilters,
-    onChange: onChangeTypeStory,
+    onChange: () => null,
```

Comment:
why is the default null instead of the default `undefined`?

I get that it's better, but by defau8lt, doest onChange expect an undefined (if it were typed)